model/types.go

```diff
@@ -298,9 +300,23 @@ type Hosts []*Host
 
 // FilesystemOpts to store fsType, fsMode, fsOwner options
 type FilesystemOpts struct {
-	Type  string
-	Mode  string
-	Owner string
+	Type       string
+	Mode       string
+	Owner      string
+	CreateOpts string
+}
+
+// GetCreateOpts returns a clean array that can be passed to the command line
+func (f FilesystemOpts) GetCreateOpts() []string {
+	cleanCharRe := regexp.MustCompile(`[^a-zA-Z0-9=, \-]`)
+	singleSpacesRe := regexp.MustCompile(`\s+`)
```

Comment:
Maybe it's just me but it's not obvious that Re on the end means Regex.  Can we just make this cleanCharRegex?

Reply:
Sure.  Code change here: https://github.com/hpe-storage/common-host-libs/pull/92